src/app/loader/loader.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, OnInit } from '@angular/core';
+import { NgxUiLoaderService } from 'ngx-ui-loader';
+
+@Component({
+  selector: 'app-loader',
+  templateUrl: './loader.component.html',
+  styleUrls: ['./loader.component.css'],
+})
+export class LoaderComponent implements OnInit {
+  constructor(private ngxLoaderService: NgxUiLoaderService) {}
+
+  ngOnInit(): void {
+    this.startLoader();
+
+    setTimeout(() => {
```

Comment:
do not laod the loader for static time , it should be be dependent on Api it self.